I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 473, in script
    jump escena_comun4
ScriptError: could not find label 'escena_comun4'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 473, in script
    jump escena_comun4
  File "C:\Program Files\renpy-8.1.3-sdk\renpy\ast.py", line 1992, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Program Files\renpy-8.1.3-sdk\renpy\script.py", line 1013, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'escena_comun4'.

Windows-10-10.0.19041 AMD64
Ren'Py 8.1.3.23091805
Dish Panic 1.0
Sun Nov  5 22:16:38 2023
